### ISSUE TITLE
lldb/python:  error: use of undeclared identifier '_Py_IsFinalizing' on Ubuntu bionic (py3.6)

### ISSUE THREAD

**Issue by sylvestre** [llvm/llvm-project#53291]
On Ubuntu bionic with Python 3.6:
```

In file included from /build/llvm-toolchain-snapshot-14~++20220119085320+7af959673e67/lldb/source/Plugins/ScriptInterpreter/Python/SWIGPythonBridge.cpp:17:
In file included from /build/llvm-toolchain-snapshot-14~++20220119085320+7af959673e67/lldb/source/Plugins/ScriptInterpreter/Python/SWIGPythonBridge.h:21:
/build/llvm-toolchain-snapshot-14~++20220119085320+7af959673e67/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h:244:11: error: use of undeclared identifier '_Py_IsFinalizing'
      if (_Py_IsFinalizing()) {
          ^
1 error generated.
```

Full log:
https://llvm-jenkins.debian.net/job/llvm-toolchain-bionic-binaries/architecture=arm64,distribution=bionic,label=arm64/221/consoleFull

**Comment by labath** [comment]
Fixed with 6ff4af8e1823337.

**Comment by sylvestre** [comment]
thanks!
